test/boulder-config-next.json

```diff
@@ -20,7 +20,7 @@
     "shutdownStopTimeout": "10s",
     "shutdownKillTimeout": "1m",
     "subscriberAgreementURL": "http://boulder:4000/terms/v1",
-    "debugAddr": "localhost:8000",
+    "debugAddr": "0.0.0.0:8000",
```

Comment:
I'll need to adjust the new configs accordingly if this lands before #1973 #1974 #1975 #1978 #1979